src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
+        using namespace jtx;
+        Env env(*this);
+        Account gw {"gw"};
+        auto wsc = makeWSClient(env.app().config());
+        env.fund(XRP(20000), "alice", "bob", gw);
+        env.close();
+        auto USD = gw["USD"];
+
+        Json::Value books;
+        {
+            books[jss::books] = Json::arrayValue;
+            {
+                auto &j = books[jss::books].append(Json::objectValue);
+                j[jss::snapshot] = true;
+                j[jss::taker_gets][jss::currency] = "XRP";
+                j[jss::taker_pays][jss::currency] = "USD";
+                j[jss::taker_pays][jss::issuer] = gw.human();
+            }
+
+            auto jv = wsc->invoke("subscribe", books);
+            BEAST_EXPECT(jv[jss::status] == "success");
```

Comment:
ok - I will add this exit-on-subscribe-error to the new tests as well as the existing tests in this file.